programming/sed-article.md

```diff
@@ -673,18 +657,18 @@ substitutions will be made or the lines that will be deleted.
 
 There are many kinds of supported addresses including:
 
-- a line number  
```

Comment:
Including two spaces at the end of a line is a markdown thing to force the next line to stay on its own line. I did that on purpose in several places.

Reply:
Got it.